src/controller_adapters/base_gamepad_adapter.py

```
@@ -0,0 +1,61 @@
+)
+
+
+class BaseGamepadTelloAdapter(TelloController):
+    """Shared button/axis-mapping logic for the standard gamepad layout.
+
+    Every supported controller (Xbox, Xbox One, GC102, Logitech F710,
+    TectInter) exposes the same `get_state()` shape via its `PyGameController`,
+    so the Tello button/axis mapping below is identical across adapters.
+    """
+
+    def __init__(self, controller):
```

Ruff (0.16.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)